src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -138,33 +138,19 @@ public void Add(ITagHelper tagHelper)
         }
 
         /// <summary>
-        /// Tracks the minimized HTML attribute.
-        /// </summary>
-        /// <param name="name">The minimized HTML attribute name.</param>
-        public void AddMinimizedHtmlAttribute(string name)
```

Comment:
As discussed offline, this method really has no reason for being anymore. Minimized tag helper attributes are always handled using the preallocated code path.